### ISSUE TITLE
more effect settings

### ISSUE THREAD

**Issue by SupinePandora43** [Nak2/StormFox#15]
hello i have problems with skybox. when option 'disable all effects' disabled i have white skybox, if option enabled, i have normal skybox, work sun, moon, but without raindrops

**Comment by Nak2** [comment]
Did 1.6 fix this?

**Comment by SupinePandora43** [comment]
no, can StormFox conflict with LifeMod? (i have LifeMod only for lifeHUD)
